### PR TITLE
Memphis.dev README

### PR DESCRIPTION
**Pull request by yanivbh1**

### CODE REVIEW

File: memphis.dev/README.md
Comment (hilafish) [inline memphis.dev/README.md]: @yanivbh1 seems like it's a new file. Need to make sure there's only one README.md in the folder.
Please keep the one above this comment (content we reviewed together) and remove the one below the comment.
Also, the banner should be used like this-
`PR <3 Memphis.dev
`
Once we push the banner to the repo, we'll approve and merge the PR (after making sure these 2 changes introduced) and then it'll be completed :)

Comment (yanivbh1) [reply memphis.dev/README.md]: @hilafish it's strange, I fetched all changes before the PR and still see only one file.
Regarding the banner - 404. File not found.

Comment (hilafish) [reply memphis.dev/README.md]: Yeah, I know :) I wrote "Once we push the banner to the repo, we'll approve and merge the PR" - and then it won't be 404. but the file name and path is already known so you can use it in the README file.

Comment (hilafish) [reply memphis.dev/README.md]: Seems like I can delete the file from here, so add the change of the banner to the README.md, and then i'll delete the redundant file

Comment (yanivbh1) [reply memphis.dev/README.md]: @hilafish I understood the issue. Should be ok now. Just need to add the banner.

File: Memphis.dev/README.md
Comment (hilafish) [inline Memphis.dev/README.md:1]: @yanivbh1 Please add on top -

# Memphis.dev & Open-Source
`PR <3 Memphis.dev`

See for reference-
https://github.com/Pull-Request-Community/open-source-rel/edit/main/Wix.com/README.md

Yes, the banner doesn't exist yet but we know upfront the banner file name and path so once it'll be ready, the PR will get approved and merged :)

Comment (hilafish) [reply Memphis.dev/README.md:1]: I'll add it :) Merging this PR.

**Review by hilafish — APPROVED** [review]

**Pull request merged**
